flexmark-html2md-converter/src/main/java/com/vladsch/flexmark/html2md/converter/internal/HtmlConverterCoreNodeRenderer.java

```diff
@@ -62,8 +62,8 @@ public Set<HtmlConverterPhase> getHtmlConverterPhases() {
     }
 
     @Override
-    public Set<HtmlNodeRendererHandler<?>> getHtmlNodeRendererHandlers() {
-        HashSet<HtmlNodeRendererHandler<? extends Node>> result = new HashSet<>(Arrays.asList(
+    public List<HtmlNodeRendererHandler<?>> getHtmlNodeRendererHandlers() {
+        List<HtmlNodeRendererHandler<? extends Node>> result = new ArrayList<>(Arrays.asList(
```

Comment:
Why do you create a new ArrayList of an existing one?
```suggestion
        List<HtmlNodeRendererHandler<? extends Node>> result = Arrays.asList(
```

Reply:
Good point, thanks :)

Reply:
In this case it has to be like this, yes